students/shirin_ak/Session09/cli_main.py

```diff
@@ -0,0 +1,100 @@
+
+import sys
+import math
+from donor_models import Donor
+from donor_models import DonorCollection
+
+"""This is the command line interface for the mailroom.
+   All the input()function calls and print() function will
+   include in this class. In other words this class will
+   handle each of mode of the program
+ """
+
+
+dc = DonorCollection()
+
+
+class CommandInterface():
+    PROMPT = "\n".join(("Please select a menu option below:",
+                    "1 - Send a Thank You to an individual",
+                    "2 - Create a Report",
+                    "3 - Send letters to all donors",
+                    "4 - Quit",
+                    ">>> "))
+
+
+
+    def thank_you_message():
+         while True:                         
+             name = input("Enter 'name' to add new donor\n"
+                         " or Enter 'list' to see list of donors\n"
+                         " (or type 'menu to exit)>>").strip()             
+
+             if name == 'list':                
+                print(print_donor_list())
+             elif name == 'menu':
+                 return
+             else:
+
+                break
+
+         while True:
+             donation_input = input("Enter the amount donated "
+                              "(or 'menu' to exit)>").strip()
+             if donation_input == 'menu':
+
+                 return
+             try:
+               amount = float(donation_input)
+               return donation_input
```

Comment:
This _return_ will prevent the rest of the code from running, so the new donor won't be added.